api/CloseEvent.json

```diff
@@ -102,6 +102,54 @@
           }
         }
       },
+      "code": {
+        "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/web-sockets.html#dom-closeevent-code",
+          "support": {
+            "chrome": {
+              "version_added": "15"
+            },
+            "chrome_android": {
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "8"
+            },
+            "firefox_android": {
+              "version_added": "8"
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "15"
```

Comment:
It seems incredibly suspect that the event interface itself is marked as supported from Opera 12.1 but none of the members are. That seems implausible. Can you do some testing in Presto, maybe add an instance to custom tests?

Reply:
I've set these to ranged values for the time being since Opera isn't one of the eight main browsers BCD is concerned about.  I'll come back to it another time!